jdi-light-bdd-tests/src/test/resources/features/FileInput.feature

```diff
@@ -0,0 +1,23 @@
+Feature: File Input element functionality
+
+  Scenario: Upload file by enabled file input element
+    Given I open "Html5 Page"
+    Then "Avatar" label text equals to "Profile picture:"
+    And "Avatar" label text contains "picture"
+    And "Avatar" is enabled
```

Comment:
remove this basic validations
And add step with all basic validations

Reply:
fixed